flavours/oauth/README.md

```diff
@@ -0,0 +1,51 @@
+# Dummy OAuth konnector
+
+This konnector is aimed to make development for OAuth konnectors easier. It uses a dedicated cozy instance as OAuth provider.
+
+Typical usage is :
+> From `cozy.tools:8080`, the Dummy OAuth konnctor make an OAuth connection to `cozy.tools:8181`.
+
+## Set up
+
+To be used, this konnector needs a little set up. We assume that there is already a CouchDB running, and a cozy stack with an instance `cozy.tools:8080`.
+
+### Dedicated instance
+
+We first need to run another cozy stack on port 8181 and create a new instance.
+
+To avoid authentication conflicts with cookies, we name our new instance `cozy.local:8181`. It means that we need to map the domain `cozy.local` to our local host. In `/etc/hosts`, just add this line:
+
+```
+127.0.0.1    cozy.local
+```
+
+Then run a new stack and create the instance
+
+```
+cozy-stack serve --port 8181 --admin-port 6161 --apps home,settings,store
+cozy-stack instances add --dev --passphrase cozy cozy.local:8181
+```
+
+### Registering OAuth client
+
+For more detail about OAuth workflow in Cozy, see https://docs.cozy.io/en/cozy-stack/auth/
+
+Then we must register our konnector on `cozy.tools:8181`.
+
+```
+curl -i -H "Accept: application/json" -H "Content-Type: application/json" -X POST -d '{"client_name":"Dummy OAuth","software_id":"cozy-konnector-dummy-oauth","redirect_uris":["http://cozy.tools:8080/accounts/dummy-oauth/redirect"]}' http://cozy.local:8181/auth/register
+```
+
+We then use the values in the responses to configure our account type in CouchDB. Let's keep `client_id`, `client_secret`
+
+First let's ensure that the `secret` database exists.
+```
+curl -X PUT localhost:5984/secrets%2Fio-cozy-account_types
+```
+
+Then let's create a document for the account type `dummy-oauth`.
+
+```
+curl -X PUT localhost:5984/secrets%2Fio-cozy-account_types/dummy-oauth \
+-d '{"grant_mode": "authorization_code","client_id": "<CLIENT_ID>","client_secret":"<CLIENT_SECRET>","auth_endpoint": "http://cozy.local:8181/auth/authorize"}'
+```
```

Comment:
Maybe you can add a short paragraph to explain how to install and use this connector (what are the fields when configuring the konnector in home for example).

Reply:
@nono Updated